src/parser.js

```diff
@@ -43,6 +43,21 @@ function parse(text, _parsers, options) {
         );
       }
     },
+    ModifierDefinition(ctx) {
+      if (!ctx.parameters) {
+        ctx.parameters = [];
```

Comment:
Why doesn't `eslint` warn about `no-param-reassign` here like in line 56?

Reply:
not sure, maybe because of this? https://eslint.org/docs/rules/no-param-reassign#options

Reply:
it's an exception of `airbnb-base`
https://github.com/airbnb/javascript/blob/f5c14cae2ff58000cead98290b8ec4b54dda2f14/packages/eslint-config-airbnb-base/rules/best-practices.js#L237